CMakeLists.txt

```diff
@@ -207,60 +220,66 @@ if(USE_TENSORRT)
          ${TVM_SRC}/src/relay/op/tensor/transform.cc
          ${TVM_SRC}/src/relay/op/tensor/reduce.cc
          ${TVM_SRC}/src/relay/op/image/resize.cc
+         ${TVM_SRC}/src/node/container.cc
+         ${TVM_SRC}/src/ir/attrs.cc
 
          ${TVM_SRC}/src/relay/ir/expr_functor.cc
          ${TVM_SRC}/src/relay/ir/pattern_functor.cc
-         ${TVM_SRC}/src/relay/pass/util.cc
-         ${TVM_SRC}/src/relay/ir/type_functor.cc
-         ${TVM_SRC}/src/relay/ir/type.cc
-         ${TVM_SRC}/src/lang/ir.cc
-         ${TVM_SRC}/src/lang/expr.cc
+         ${TVM_SRC}/src/relay/op/type_relations.cc
```

Comment:
Can we use GLOB here to get the files? It looks not very safe to add them one by one IMH.

Reply:
I would prefer to use glob as well. I did that originally but there was some concern about increasing the size of the binary. I think the size was around 7mb with glob and ~3mb with this approach. It is quite a painstaking process to pick out the files one by one.

Reply:
NVM, we can keep it for now. We will need to work on the JSON runtime later. Once that it ready, we should be able remove most of them (probably even all) as only runtime data structures and files are needed.